README.md

```diff
@@ -264,11 +266,65 @@ The recommended strategy for Heroku is to setup a
 *:information_desk_person: Note: for high volume apps Heroku log drains will drop messages. This
 is true for any Heroku app, in which case we recommend the Network method below.*
 
+---
+
 </p></details>
 
 <details><summary><strong>All other platforms (Network / HTTP)</strong></summary><p>
 
-Coming soon!
+Timber does *not* force an HTTP client on you. The following instruction utilize the Timber default
+`Timber.Transports.HTTP.HackneyClient`. This is a highly efficient client that utilizes hackney,
+batching, stay alive connections, connection pools, and msgpack to deliver logs with high
+throughput and little overhead. If you'd like to use another client see
+`Timber.Transports.HTTP.Client`.
+
+1. *Add* HTTP dependencies to `mix.exs`:
+
+  ```elixir
+  def application do
+    [applications: [:hackney, :timber]] # <-- Be sure to add hackney!
+  end
+
+  def deps do
+    [
+      {:timber, "~> 1.0"},
+      {:hackney, "~> 1.6"} # <-- ADD ME
+    ]
+  end
+  ```
+
+2. *Configure* Timber to use the Network transport in `config/prod.exs`:
+
+  ```elixir
+  # config/prod.exs
+
+  config :timber,
+    transport: Timber.Transports.Network,
+    api_key: System.get_env("TIMBER_API_KEY")
+  ```
+
+3. *Start* the Timber modules in `myapp.ex`:
+
+  ```elixir
+  # myapp.ex
+
+  def start(_type, _opts) do
+    :ok = Timber.Transports.HTTP.HackneyClient.start() # <-- ADD ME
+
+    children = []
+    opts = [strategy: :one_for_one, name: MyApp.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+  ```
+
+4. Obtain your Timber API :key: by **[adding your app in Timber](https://app.timber.io)**.
+   Afterwards simply assign it to the `TIMBER_API_KEY` environment variable.
+
+* Note: we use the `Network` transport so that we can upgrade protocols in the future if we
+  deem it more efficient. For example, TCP. If you want to use strictly HTTP, simply replace
+  `Timber.Transports.Network` with `Timber.Transports.HTTP`.
```

Comment:
Good points. My intent was leave the door open to non-batch writing. For example, with TCP we could write to the socket for every line versus buffering / batching. This would have a more "real time" feel, but we currently do not support TCP.